pkgs/stdenv/generic/make-derivation.nix

```diff
@@ -7,7 +9,7 @@ let
     # to build it. This is a bit confusing for cross compilation.
     inherit (stdenv) hostPlatform;
   };
-in rec {
+in
   # `mkDerivation` wraps the builtin `derivation` function to
```

Comment:
This comment / documentation placement is confusing now...

Reply:
I was thinking of fixing that in a separate commit to help with git blame, but I should make it part of this PR yes.